apps/typegpu-docs/src/components/ExampleView.tsx

```diff
@@ -66,29 +66,35 @@ function useExample(
   }, [exampleCode, htmlCode, setSnackbarText, setExampleControlParams]);
```

Comment:
`exampleCode` is not used in this useEffect, while `tsSources` is.

```suggestion
  }, [tsSources, htmlCode, setSnackbarText, setExampleControlParams]);
```